convert_hf_to_gguf.py

```diff
@@ -7186,6 +7208,48 @@ def modify_tensors(self, data_torch: Tensor, name: str, bid: int | None):
 
         return super().modify_tensors(data_torch, name, bid)
 
+@ModelBase.register("PanguEmbeddedForCausalLM")
+class PanguEmbeddedModel(TextModel):
+    model_arch = gguf.MODEL_ARCH.PANGU_EMBED
+    
+    def set_vocab(self):
+        try:
+            self._set_vocab_pangu_embedded()
+        except FileNotFoundError:
+            print("pangu vocab set fail, fallback to sentencepiece!")   
+            self._set_vocab_sentencepiece()
```

Comment:
```suggestion
        self._set_vocab_sentencepiece()
```
I'm pretty sure this will work just fine.